domains/primitives/auto-id/src/runtime_interface.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(feature = "std")]
+use crate::host_functions::HostFunctionExtension;
+use crate::{DerVec, SignatureVerificationRequest, TbsCertificate};
+#[cfg(feature = "std")]
+use sp_externalities::ExternalitiesExt;
+use sp_runtime_interface::runtime_interface;
+
+/// Signature verification runtime interface.
+#[runtime_interface]
+pub trait SignatureVerificationRuntimeInterface {
```

Comment:
Does the name `SignatureVerificationRuntimeInterface` still make sense given we have added `decode_tbs_certificate`?